companies-manifest.js

```diff
@@ -647,6 +648,25 @@ const companies = [
     },
     projects: {},
   },
+  {
+    key: 'https://chemondis.com/',
+    similarWebGlobalRank: 404819,
+    name: 'CheMondis',
+    logo: CheMondisLogo,
+    style: {
+      bottom: '-0.2rem',
+    },
+    projects: {
+      marketplace: {
+        title: 'CheMondis - Marketplace',
```

Comment:
I'm not a big fan of having the name of the brand alongside the name of the project, it's something that I'm looking forward to removing from some other items that we have on the showcase (Looking at you AirBnB Cereal) since it's kinda redundant now that we have the company name right below the project name. On the other hand I must admit that 
"Marketplace"  would end up being quite generic as a name so I'll leave it up to you.
In the case you decide to leave it as is, can you please get rid of the Hyphen though?

Reply:
Great feedback and will proceed with the changes 👍 , just followed the pattern of the others 😊 . 
About not being displayed due to the low similarweb ranking any plans to show all in the future? 🤔

Reply:
I'm actually working on an alternative grid view to accompany the slideshow in order to make sure more websites get their space in the spotlight. Hopefully that's going to help in the beginning :)

Reply:
@el1f done 😄